scripts/network-launch/launch.sh

```diff
@@ -33,26 +33,40 @@
 #path="../../root/"
 
 #### TESTNET ####
-bootNodeIP=165.232.149.206
-bootNodeHost=mainnet-node-1.stage.cere.network
-genesisValidatorIP=45.55.62.114
-genesisValidatorHost=mainnet-node-2.stage.cere.network
-validatorsIPs=(159.203.13.16\
-               157.230.106.171)
-validatorsHosts=(mainnet-node-3.stage.cere.network\
-                 mainnet-node-4.stage.cere.network)
-fullNodeIP=143.198.130.232
-fullNodeHost=rpc.mainnet.stage.cere.network
-archiveNodeIP=178.128.250.106
-archiveNodeHost=archive.mainnet.stage.cere.network
+#bootNodeIP=165.232.149.206
```

Comment:
I suppose this is for testing? 
This is why we don't remove commented stuff yet, right?

Reply:
This is for possible future nodes updates/relaunches. A good idea will be to move each env to a separate file and include it as a parameter. Will create a task.